styles/StyledDiv.js

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components';
+
+const StyledFlex = styled.div`
```

Comment:
The naming confuses me: There is  no `dispay: flex` in here.

Reply:
absolutly right. I renamed the const accordingly.